packages/usa-tooltip/src/index.js

```diff
@@ -394,6 +393,9 @@ const tooltip = behavior(
         window.removeEventListener("keydown", handleEscape);
       },
     },
+      keydown: {
+        [BODY]: keymap({ Escape: handleEscape})
+      },
```

Comment:
Thanks for making this work with our more standard code patterns, @mahoneycm. 

One thing I noticed with this method is that the page registers `escape` presses even when the tooltip is inactive. What I liked about manually adding and removing the `eventlistener` on mouse over/out or focus in/out is that the window only would register the key presses when the tooltip is active. I feel like this would limit unintended interaction with other components that also rely on the `escape` keypress. Curious what you think!

Reply:
That's a great point! To be honest I'm not too familiar with the benefit/loss of having a persistent listener but I understand the principle of only listening for the keypress while the tooltip is active!

I considered having an early return in `handleEscape` if there were no active tooltips but that doesn't change the fact that the listener will be there the entire time.

Happy to move forward with the other solution if we feel like there is additional value there!